### ISSUE TITLE
What is the meaning of to_skip list when I using ranking method in tkbc?

### ISSUE THREAD

**Issue by ArchyZheng** [czy1999/MultiTQ#2]
Hi,

This is a very wonderful job you did. When I try to train the embedding by tkbc, the meaning of `to_skip` list used in `ranking` method is very confusing.

Can you tell me more detail about that?

Thank you!

**Comment by czy1999** [comment]
Hi there,

Thank you for reaching out and for your kind words!

Regarding your query about the 'to_skip' list in the ranking method: 'to_skip' is an part of computing filtered metrics. You can find an explanation for this in line 24 of the 'process_icews.py' file. Essentially, 'to_skip' contains a dictionary of quadruples from the training set. These are used to ensure that when evaluating the model on the validation or test set, any positive triples that also appear in the training data are not considered. This helps in accurately assessing the model's ability to predict new, unseen facts.

I hope this clears up any confusion. If you have further questions or need more detailed explanations, feel free to ask!

Best regards.

**Comment by ArchyZheng** [comment]
Thanks for your reply! I can understand the meaning of the special list very well according your explanation.  

By the way, happy new year!

Best regards.
